Client/Coordinators/TabTray/RemoteTabsCoordinator.swift

```diff
@@ -0,0 +1,55 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class RemoteTabsCoordinator: BaseCoordinator, RemoteTabsPanelDelegate {
+    // MARK: - Properties
+    weak var parentCoordinator: ParentCoordinatorDelegate?
+    private let profile: Profile
+    private var fxAccountViewController: FirefoxAccountSignInViewController?
+    private var applicationHelper: ApplicationHelper
+    weak var qrCodeNavigationHandler: QRCodeNavigationHandler?
```

Comment:
nit, we could put the delegates together
```suggestion
    private let profile: Profile
    private var fxAccountViewController: FirefoxAccountSignInViewController?
    private var applicationHelper: ApplicationHelper

    weak var qrCodeNavigationHandler: QRCodeNavigationHandler?
    weak var parentCoordinator: ParentCoordinatorDelegate?
```